apps/dokploy/templates/teable/docker-compose.yml

```diff
@@ -0,0 +1,83 @@
+version: '3.9'
+
+services:
+  teable:
+    image: ghcr.io/teableio/teable:1.3.1-alpha-build.460
+    restart: always
+    ports:
+      - ${TEABLE_PORT}
+    volumes:
+      - teable-data:/app/.assets:rw
```

Comment:
Remove the `:rw`, if you deploy and try to deploy again it will become unnaccesible and it will say bad gateway but deleting that will remove the issue